index.js

```diff
@@ -595,90 +595,43 @@ function initFirstApp(configObject, configUrl, licenseKey) {
 }
 
 function registerShortcuts() {
-    const resetZoomShortcut = 'CommandOrControl+0';
-    const zoomInShortcut = 'CommandOrControl+=';
-    const zoomInShiftShortcut = 'CommandOrControl+Plus';
-    const zoomOutShortcut = 'CommandOrControl+-';
-    const zoomOutShiftShortcut = 'CommandOrControl+_';
-    const devToolsShortcut = 'CommandOrControl+Shift+I';
-    const reloadF5Shortcut = 'F5';
-    const reloadShiftF5Shortcut = 'Shift+F5';
-    const reloadCtrlRShortcut = 'CommandOrControl+R';
-    const reloadCtrlShiftRShortcut = 'CommandOrControl+Shift+R';
-
-    let zoom = (zoomIn, reset = false) => {
-        return () => {
-            let browserWindow = BrowserWindow.getFocusedWindow();
-            let windowOptions = browserWindow && coreState.getWindowOptionsById(browserWindow.id);
-
-            if (windowOptions && windowOptions.accelerator && windowOptions.accelerator.zoom) {
-                browserWindow.webContents.send('zoom', zoomIn, reset);
-            }
-        };
-    };
+    app.on('browser-window-focus', (event, browserWindow) => {
+        const windowOptions = coreState.getWindowOptionsById(browserWindow.id);
+        const accelerator = windowOptions && windowOptions.accelerator || {};
+        const webContents = browserWindow.webContents;
 
-    let reload = () => {
-        let browserWindow = BrowserWindow.getFocusedWindow();
-        let windowOptions = browserWindow && coreState.getWindowOptionsById(browserWindow.id);
+        if (accelerator.zoom) {
+            const zoom = zoomIn => { return () => { webContents.send('zoom', zoomIn, zoomIn === undefined); }; };
```

Comment:
This can be refactored. Instead of sending zoom levels to the renderer to deal with, we can handle everything on the browser side by calling `browserWindow.webContents.getZoomLevel()` and `browserWindow.webContents.setZoomLevel()`.

https://github.com/HadoukenIO/core/blob/develop/src/renderer/api-decorator.js#L175

Reply:
This could also be moved to another ticket/PR. I'll leave it up to you.

Reply:
As discussed we won't do your suggested refactor as the Chromium so-called "by design" behavior [is undesirable](https://bugs.chromium.org/p/chromium/issues/detail?id=390775).